themes-default/slim/views/config_backuprestore.mako

```diff
@@ -21,8 +21,60 @@ const startVue = () => {
         },
         data() {
             return {
-                header: 'Backup/Restore'
+                header: 'Backup/Restore',
+                backup: {
+                    disabled: false,
+                    status: '',
+                    dir: ''
+                },
+                restore: {
+                    disabled: false,
+                    status: '',
+                    file: ''
+                }
             };
+        },
+        mounted() {
+            $('#backupDir').fileBrowser({
+                title: 'Select backup folder to save to',
+                key: 'backupPath'
+            });
```

Comment:
The file browser doesn't exactly work.
They open, you can select a folder, but the value in the Vue instance isn't updated.
I think you either have to use the `file-browser` component, or not convert that part to Vue... :expressionless:

Reply:
We can't use the `file-browser` component because the component doesn't work when there's more than one file browser on the page.
To be able to use that we'd need to completely redo file-browser without jQuery.
I'll try to find a workaround for this page, but I'll probably have to drop the Vue integration to fix it.